varia/functions.php

```diff
@@ -225,15 +225,16 @@ function varia_setup() {
 				'enable_theme_default' => true,
 			)
 		);
-		
+
 		// Add support for Content Options.
 		add_theme_support( 'jetpack-content-options', array(
 			'blog-display' => 'content',
 			'featured-images' => array(
-				'archive'         => true,
-				'archive-default' => true,
-				'post'            => true,
-				'page'            => true,
+				'archive'         => false,
+				'archive-default' => false,
+				'post'            => false,
+				//featured images on pages boolean was previously managed by Varia rather than jetpack.  If that value has been set default to that.
+				'page'            => get_theme_mod( 'show_featured_image_on_pages', false ),
```

Comment:
As I understand it, this would remove that option when you're only seeking to set the default value: https://jetpack.com/support/content-options/

```suggestion
				'page-default'    => get_theme_mod( 'show_featured_image_on_pages', false ),
```

You could add `post-default` as `false` too, if you'd prefer.

Reply:
Looking at the existing behavior it looks like all instances DO display the featured image, so (except for pages when had already introduced that option defaulting to false) the rest I believe are correct defaulting to 'true'.